src/assets/css/themes/darkly-red.css

```diff
@@ -3721,6 +3723,9 @@ input[type="submit"].btn-block {
   pointer-events: none;
   cursor: default;
 }
+.nav-link .nav-link__icon-text {
```

Comment:
Is nav-link__icon-text used in places other than as a descendant of a nav-link? If so, you might be able to apply it directly.

Reply:
I wanted to add proper specifity for this one, so that no one will be able to use this class outside `.nav-link`